src/Mod/Sketcher/Gui/CommandSketcherTools.cpp

```diff
@@ -876,7 +928,7 @@ void CmdSketcherSelectElementsAssociatedWithConstraints::activated(int iMsg)
 
 bool CmdSketcherSelectElementsAssociatedWithConstraints::isActive()
 {
-    return isCommandActive(getActiveGuiDocument(), true);
+    return isCommandActive(getActiveGuiDocument(), true, false, false, false, false, false, true);
```

Comment:
This call is quite cryptic with this many bools as parameters. It's really difficult to know what the seventh bool is for.